src/System.Numerics.Tensors/System/Numerics/DenseTensor.cs

```diff
@@ -98,6 +98,29 @@ public override void SetValue(int index, T value)
             Buffer.Span[index] = value;
         }
 
+        protected override void CopyTo(T[] array, int arrayIndex)
+        {
```

Comment:
Do you want to do any argument validation here?

Reply:
Yes, will add.